web/styles/app_variables.css

```diff
@@ -61,6 +61,11 @@
     /* Tippy popover related values */
     --popover-menu-min-width: 230px;
 
+    /* Information density and typography values */
+    /* The legacy values here are updated via JavaScript */
+    --base-font-size-px: 14px;
+    --base-line-height-unitless: calc(20 / 14);
```

Comment:
The legacy value is `calc(17 / 14)`, not `calc(20 / 14)`.

Reply:
You're right. I'll fix that in the PR that starts putting those values to use.